generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
+
+	"github.com/shurcooL/go/vfs/httpfs/filter"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	ignore := func(fi os.FileInfo, path string) bool {
+		switch {
+		case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
+			return true
+		case pathpkg.Ext(fi.Name()) == ".go":
+			return true
+		case path == "/README.md":
+			return true
+		default:
+			return false
+		}
+	}
+	var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
+
+	err := vfsgen.Generate(data, vfsgen.Options{
+		Filename:     "data.go",
+		PackageName:  "data",
+		VariableName: "Assets",
+	})
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
> I'm not against reducing the verbosity, I just wanted to point out that trying to match the previous line count may not be an optimal goal.

100% agree.

> I have thought about it, and I think the command case is more common use case. It stands to reason to have the defaults optimized for that case, as they currently are. That's why I don't think this change can be made. Given that appdash-data is a more advanced use, it seems normal to have it supply custom options.

That makes sense. But we can still drop Filename and VariableName, right?

> it will generate output in a standard Assets_vfsdata.go location.

Surely it's (lowercase) `assets_vfsdata.go`, right? Uppercase seems pretty non-standard naming convention for source code files.

> I would move all assets that are meant to be embedded inside a single subdirectory, for example named assets (or data).

That looks great -- yes! I'm not certain of the naming though, which is more consistent assuming we don't specify `VariableName`?